src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 20;
+    private static final int MAX_VALUE_OF_DEFAULT_FIGURE = 10;
+    private static final String COLOR_OF_DEFAULT_FIGURE = "White";
+    private static final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure(Figure[] figures) {
+        int index = random.nextInt(figures.length);
+
+        switch (index) {
+            case 0:
+                double radius = random.nextDouble(MAX_VALUE);
+                String color = colorSupplier.getRandomColor();
+                return new Circle(radius, color);
+            case 1:
+                double firstLeg = random.nextDouble(MAX_VALUE);
```

Comment:
create separate method for `random.nextDouble(MAX_VALUE)` and use it for each type of figure